src/lib/components/ui/ButtonGroup/README.mdx

```diff
@@ -0,0 +1,206 @@
+---
+name: ButtonGroup
+menu: 'Components: UI'
+route: /components/ui/button-group
+---
+
+# ButtonGroup
```

Comment:
It shold say, that in case I want to include `Button` in `ButtonGroup` conditionally, I need to either include the `Button` or `undefined`.

Reply:
Like this? But why? `&&` also works…

```jsx
<ButtonGroup>
  {condition ? (<Button variant="dark" label="Week" />) : undefined}
  <Button label="Month" />
  <Button label="Year" />
</ButtonGroup>
```

Reply:
> Like this?

yes

> But why? && also works

I just tested it and passing in `false` instead of a child element causes a propType warning in console. Passing in `null` or `undefined` works.

Therefore the syntax `condition && <Button />` as long as `condition` does not evaluate to `false`. I think this needs to be mentioned.

Reply:
Ah, I see. But then it will be a problem of all props that we validate as `node` which is by far not just the `ButtonGroup`. One part of me says that `propTypes` should tell enough and the other that we could mention it, but in all such components or somewhere globally. I'm taking down a note on this and closing it for now.